testing tool/grpc-test-example/basic-grpc-server-testing/testing-tool/Readme.md

```diff
@@ -0,0 +1,89 @@
+[15:32] Jonty Esterhuizen
+
+# TMS-Service
+
+The TMS service is a NodeJs REST-API service that ingests a FRM transaction and validates if it has the correct format.
```

Comment:
It's not a REST service. It's a gRPC service that using a single REST endpoint to be activated. (or better explanation)